testreport/tasks.py

```diff
@@ -147,3 +153,35 @@ def update_state(bug):
             get_issue_fields_from_bts(bug.externalId)['status']['name']
         log.debug('Saving bug "{}"'.format(bug.externalId))
         bug.save()
+
+
+@celery.task()
```

Comment:
This operation can fail for different reason, network, s3 provider fail, removed/renamed/moved files etc.
Lets add retry count for task and timeout between retries for celery task.